src/mercator/mercator/static/js/Packages/MercatorProposal/Detail.html

```diff
@@ -7,7 +7,7 @@
     }">
     <h2 class="print-only section-jump-cover-header" data-aria-hidden="true">{{ data.title }}</h2>
     <div class="section-jump-cover">
-        <ul class="meta-list mercator-proposal-detail-meta-list">
+        <ul class="meta-list mercator-proposal-detail-meta-list m-phase-{{data.currentPhase}}">
```

Comment:
Perhaps this should be numeric in case we change the phase names. Fred already told that what we call "participate" they call "join"